assets/js/blocks/mini-cart/edit.tsx

```diff
@@ -56,32 +50,35 @@ interface Props {
 	setProductCountColor: ( colorValue: string | undefined ) => void;
 }
 
-const Edit = ( {
-	attributes,
-	setAttributes,
-	clientId,
-	setPriceColor,
-	setIconColor,
-	setProductCountColor,
-}: Props ): ReactElement => {
+const Edit = ( { attributes, setAttributes }: Props ): ReactElement => {
 	const {
 		cartAndCheckoutRenderStyle,
 		addToCartBehaviour,
 		hasHiddenPrice,
-		priceColorValue,
-		iconColorValue,
-		productCountColorValue,
+		priceColor = defaultColorItem,
+		iconColor = defaultColorItem,
+		productCountColor = defaultColorItem,
 		miniCartIcon,
-	} = attributes;
+	} = migrateAttributesToColorPanel( attributes );
 
-	const className = classNames( {
-		'wc-block-mini-cart': true,
-		'has-price-color': priceColorValue,
-		'has-icon-color': iconColorValue,
-		'has-product-count-color': productCountColorValue,
-	} );
+	const miniCartColorAttributes = {
+		priceColor: {
+			label: __( 'Price', 'woo-gutenberg-products-block' ),
+			context: 'price-color',
+		},
+		iconColor: {
+			label: __( 'Icon', 'woo-gutenberg-products-block' ),
+			context: 'icon-color',
+		},
+		productCountColor: {
+			label: __( 'Product Count', 'woo-gutenberg-products-block' ),
+			context: 'product-count-color',
+		},
+	};
 
-	const blockProps = useBlockProps( { className } );
+	const blockProps = useBlockProps( {
+		className: [ 'wc-block-mini-cart' ].join( ' ' ),
+	} );
```

Comment:
This can be simplified now. :slightly_smiling_face: 
```suggestion
	const blockProps = useBlockProps( { className: 'wc-block-mini-cart' } );
```